src/trading/TakeOrder.sol

```diff
@@ -326,11 +323,10 @@ library Trade {
     }
 
     function getSharePriceDetails(IMarket _market, IOrders _orders, bytes32 _orderId) private constant returns (uint256 _sharePriceRange, uint256 _sharePriceLong, uint256 _sharePriceShort) {
-        int256 _maxDisplayPrice = _market.getMaxDisplayPrice();
-        int256 _minDisplayPrice = _market.getMinDisplayPrice();
-        int256 _orderDisplayPrice = _orders.getPrice(_orderId);
-        _sharePriceRange = uint256(_maxDisplayPrice - _minDisplayPrice);
-        _sharePriceLong = uint256(_orderDisplayPrice - _minDisplayPrice);
+        uint256 _maxDisplayPrice = _market.getMarketDenominator();
+        uint256 _orderDisplayPrice = _orders.getPrice(_orderId);
+        _sharePriceRange = _maxDisplayPrice;
+        _sharePriceLong = _orderDisplayPrice;
```

Comment:
These variable names no longer seem correct.  The first is `marketDenominator` and the second is `orderPrice`.  Also, no need for the intermediate variables here anymore I think, we can just directly assign to `_sharePriceRange` and `_sharePriceLong`.